curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/local-scope-and-functions.english.md

```diff
@@ -59,15 +61,13 @@ function myLocalScope() {
 
   // Only change code below this line
 
-  console.log(myVar);
+  console.log('inside myLocalScope', myVar);
 }
 myLocalScope();
 
 // Run and check the console
 // myVar is not defined outside of myLocalScope
-console.log(myVar);
-
-// Now remove the console log line to pass the test
+console.log('outside myLocalScope',myVar);
```

Comment:
```suggestion
console.log(myVar); // Outside myLocalScope
```
The content of this console.log will never reach the console. Either way is fine...

Reply:
It may not be necessary, since the state of https://github.com/freeCodeCamp/freeCodeCamp/pull/38040 has not been decided, but I'd prefer to keep comments to a minimum.  Each comment would be one more requiring translation.